src/openforms/registrations/contrib/stuf_zds/plugin.py

```diff
@@ -477,6 +478,58 @@ def get_config_actions(self):
             ),
         ]
 
+    def update_registration_with_confirmation_email(
+        self, submission: Submission, options: RegistrationOptions
+    ) -> dict:
+        logger.info("stuf_zds_finalise_registration_started", options=options)
+        res = get_last_confirmation_email(submission)
+        if res is None:
+            logger.warning(
```

Comment:
I think we can lower this to `info` instead of `warning`, I'd expect confirmation email failures to be logged themselves. This may also fail because there was no recipient e-mail address configured/provided.